test/service/test_manifest.py

```diff
@@ -440,14 +440,37 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                location_prefix = f'url="{config.service_endpoint()}/repository/files'
                 curl_files = []
+                urls = []
+                related_urls = []
                 for line in lines:
                     if line.startswith(file_prefix):
                         self.assertTrue(line.endswith('"'))
                         file_name = line[len(file_prefix):-1]
                         curl_files.append(file_name)
+                    elif line.startswith(location_prefix):
+                        self.assertTrue(line.endswith('"'))
+                        url = furl(line[len(location_prefix):-1])
+                        try:
+                            url.args['drsPath']
```

Comment:
I you don't need the result of the expression it is fine to use the `in` operator, instead of try: except

Reply:
`(x if k in d else y).append(...)`